app/controllers/spotlight/pages_controller.rb

```diff
@@ -116,8 +116,17 @@ def _prefixes
       @_prefixes ||= super + ['catalog']
     end
 
+    # Add a Page specific search_results method that takes user params as
+    # an option and passes that off to the search service to get results
+    # @param [Hash] the query parameters
+    # @return [Object] the search results object from the configured search service
+    def search_results(user_params)
```

Comment:
I could easily be convinced to use a method name that doesn't match the API of the search_service (like `block_search_results` or something like that), but kept it the same for now.